fs/_ftp_parse.py

```diff
@@ -82,15 +85,14 @@ def parse_line(line):
 
 
 def _parse_time(t, formats):
-    t = " ".join(token.strip() for token in t.lower().split(" "))
-
     _t = None
     for frmt in formats:
         try:
             _t = time.strptime(t, frmt)
+            break
         except ValueError:
             continue
-    if not _t:
+    else:
```

Comment:
Oooh, today I just learned about the `for ... else ... ` syntax in Python :slightly_smiling_face: